easybuild/easyconfigs/c/CUDA/CUDA-10.1.eb

```diff
@@ -0,0 +1,15 @@
+easyblock = "ModuleOnly"
+name = 'CUDA'
+version = '10.1'
+
+homepage = 'https://developer.nvidia.com/cuda-toolkit'
+description = """CUDA (formerly Compute Unified Device Architecture) is a parallel
+ computing platform and programming model created by NVIDIA and implemented by the
+ graphics processing units (GPUs) that they produce. CUDA gives developers access
+ to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs."""
+
+toolchain = {'name': 'GCC', 'version': '7.3.0'}
+
+dependencies = [ ('CUDAcore','10.1.243') ]
```

Comment:
Doesn't that need hiddendependencies?

Reply:
Mmm, well, it installed and found the dependency just fine on our system, and I don't see anything in our config.cfg related to hidden dependencies ?

Reply:
Ok, haven't used hidden_deps so don't really know hot it works.

Reply:
hiddendependencies isn't really necessary, if you use it it's site-specific, it basically enforces that the module name should have a dot, but normal dependencies will also look for hidden modules (whether dotted or configured so via Lmod config files, doesn't matter)